modules/core/src/main/java/org/apache/ignite/internal/processors/platform/utils/PlatformConfigurationUtils.java

```diff
@@ -667,9 +668,7 @@ public static void readIgniteConfiguration(BinaryRawReaderEx in, IgniteConfigura
 
         int sqlSchemasCnt = in.readInt();
 
-        if (sqlSchemasCnt == -1)
```

Comment:
That is what it does already?

Reply:
I thought that it's gonna be 0 now, that is more natural and respects current .NET changes.
see your IgniteConfiguration.cs changes - it now writes 0 instead of magic -1 in case of empty (null) sqlSchemas.

Gonna work either way. Since -1 is less or equal zero. But now Java and C# will write it differently. Will not they?

Reply:
You are right, I missed that one. Fixed, thanks.